### ISSUE TITLE
"_call": ".()"

### ISSUE THREAD

**Issue by jajajaqlt** [capergroup/bayou#196]
The resulting sketches of dom driver have 10% having "_call": ".()". An example is below. Does anybody know what really happens? This bug will crash evidence_extractor.
```
    {
      "javadoc": null,
      "sequences": [
        {
          "calls": [
            "java.util.Collection<Tau_E>.iterator()",
            ".()"
          ]
        }
      ],
      "ast": {
        "_nodes": [
          {
            "_call": "java.util.Collection<Tau_E>.iterator()",
            "node": "DAPICall",
            "_returns": "java.util.Iterator",
            "_throws": []
          },
          {
            "_call": ".()",
            "node": "DAPICall",
            "_returns": "void",
            "_throws": []
          }
        ],
        "node": "DSubTree"
      }
    },
```

**Comment by vijay-murali** [comment]
Do you have a program for which the DOM driver produces this output?

**Comment by jajajaqlt** [comment]
First, I got rid of those .()'s. I didn't keep the original java files and corresponding java.json files. So we don't know which programs produces these bugs. BTW, there are just so many of the programs having these .() outputs. This might be a problem of the dom driver.

**Comment by vijay-murali** [comment]
Fixed in 98c16c93a6b7da66a804d509a0be5f6517f7f241.
